grayskull/utils.py

```diff
@@ -108,6 +108,12 @@ def origin_is_local_sdist(name: str) -> bool:
     )
 
 
+def origin_is_tree(name: str) -> bool:
+    """Return True if it is a directory"""
+    path = Path(name)
+    return path.is_dir()
```

Comment:
We could look for pyproject.toml or setup.py; it should work with either.

Reply:
that can work, but you also need to look for ``setup.cfg``

Reply:
With the pypa/build method, pyproject.toml is the only necessary file to get the metadata.

Reply:
I agree, but not all projects uses it :/

Reply:
pypa/build works with setup.py or pyproject.toml. It asks the underlying build system to create a wheel with `package.dist-info/METADATA`. Then we read `METADATA` and not `pyproject.toml`. So with this technique we don't touch setup.py, setup.cfg or pyproject.toml; pypa/build writes METADATA and we read that.